lib/js/atoms/Modal/Modal.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+	<div class="m-modal" @click.stop="$emit('close-modal')">
+		<div class="m-modal__contentWrapper" @click.stop>
+			<div v-if="showHeader" class="m-modal__header">
+				<slot name="header" />
+				<wnl-icon
+					touchable
+					:icon="ICONS.FA_TIMES"
+					class="-stormGray m-modal__header__close"
+					:size="ICON_SIZES.SMALL"
+					@click.native.stop="$emit('close-modal')"
+				/>
+			</div>
+			<div
+				class="m-modal__content"
+				:class="{ 'scrollable-main-container': animateScrollingContent }"
```

Comment:
`scrollable-main-container` is defined in https://github.com/bethinkpl/wnl-platform/blob/4c4ea3350c83dab06d1cc5f6791d73536dcdf464/resources/assets/styles/_app.scss#L147-L162
Maybe remove it for now and figure it out when replacing old modal in platform?

Reply:
Ah, we're already replacing it in https://github.com/bethinkpl/wnl-platform/pull/3461

I'd either close eyes and copy the class or figure out if we really need it.

Reply:
Modal in DS is temporary/unofficial. UX don't like the look of this component, but we needed it here to correctly implement `SurveyQuestion`. It was just moved from platform for now.

We're going to replace existsing modal with a new one, put it into Storybook and solve this problems later

Reply:
I'd copy the class styles as a quick fix, then. Right now we have a circular dependency.

Reply:
Before I copy that, do you remember what was it required for? I see https://bethink.atlassian.net/browse/IT-2890 but I can't find what it changes.


@tweetgeek maybe you remember https://github.com/bethinkpl/wnl-platform/commit/c2c45df4c25b997953f56c661e712ae745616c93 ?

Reply:
If it was supposed to fix scrolling inside modal it isn't fixing that: https://lek.wiecejnizlek.pl/app/courses/1/lessons/34/645/81

Reply:
I have no idea, it looks the same as annotations modal to me.

Reply:
https://bethink.atlassian.net/browse/IT-2890?focusedCommentId=27350 should scroll the screen to the form when clicking "Skomentuj" in the case of a QuizQuestion reference, but actually does not :(

Reply:
Imo, this `className` doesn't solve the problem. 

See https://bethink.atlassian.net/browse/PLATFORM-1145 and https://github.com/bethinkpl/wnl-platform/pull/3492